sklego/meta/zero_inflated_regressor.py

```diff
@@ -112,9 +123,12 @@ def fit(self, X, y, sample_weight=None):
                 logging.warning("Classifier ignores sample_weight.")
                 self.classifier_.fit(X, y != 0)
 
-        non_zero_indices = np.where(y != 0)[0]
+        indices_for_training = np.where(y != 0)[0] # these are the non-zero indices
+        if (handle_zero == 'ignore') & (indices_for_training.size == 0): # if we choose to ignore that all train set output is 0
+            logging.warning("Regressor ignores that all train set output consists of zero values.")
```

Comment:
I am not fully convinced by this warning message. I think we should be more verbose in stating what's happening.

What do you think of the following:

```suggestion
            logging.warning("Regressor will be training on `y` consisting of zero values only.")
```

Reply:
Indeed, your edit is clearer, thank you for the suggestion